hooks/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker build --build-arg APIGENTOOLS_COMMIT=${SOURCE_COMMIT} -f $DOCKERFILE_PATH -t $IMAGE_NAME .
```

Comment:
Are env variables the way to go here?

Reply:
So this is actually just a modification of the dockerhub's default [1], to which I added the `--build-arg`.

[1] https://docs.docker.com/docker-hub/builds/advanced/#build-hook-examples